instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Bregma-left.jsonld

```diff
@@ -0,0 +1,46 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/PW-RBSC-corT_v2004-Bregma-left",
+  "@type": "https://openminds.ebrains.eu/core/CommonCoordinateSpaceVersion",
+  "abbreviation": "PW-RBSC-corT",
+  "accessibilty": {
```

Comment:
GitHub Actions flagged this one as a spelling error

Reply:
jup...should be "accessibility" (last i is missing...)